packages/terra-site/src/examples/table/Index.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-table/docs/README.md';
+// Component Source
+// eslint-disable-next-line import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions
```

Comment:
Instead of disabling just the next-line, eslint allows you to disable blocks of code.
```
/* eslint-disable import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
import TableSrc from '!raw-loader!terra-table/src/Table';
import SingleSelectableRowsSrc from '!raw-loader!terra-table/src/SingleSelectableRows';
import TableHeaderSrc from '!raw-loader!terra-table/src/TableHeader';
....
/* eslint-enable import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
```